### DISCUSSION TITLE
xdrip with G6 asks for calibration every day

### DISCUSSION THREAD

**Discussion by jaslo** [NightscoutFoundation/xDrip#3472] [General]
I'm running xdrip4ios with a G6 and it's asking for calibration every day. Is this normal? Is there an option somewhere to turn it off?
The readings from the sensor are quite accurate and the calibration really seems unnecessary.

**Comment by Navid200** [comment]
G6 only asks for daily calibrations if you have entered no calibration code when you started the session.

These forums are for xDrip, which is an Android app.
If you are using xDrip4ios, you need to reach the corresponding group, which is not here.
